packages/firestore/exp/index.node.ts

```diff
@@ -15,8 +15,37 @@
  * limitations under the License.
  */
 
+import { version } from '../package.json';
+import { _registerComponent, registerVersion } from '@firebase/app-exp';
+import { Component, ComponentType } from '@firebase/component';
+import { Firestore } from './src/api/database';
+
 export { FieldPath, documentId } from '../lite/src/api/field_path';
 
+export {
+  Firestore,
+  initializeFirestore,
+  getFirestore
+} from './src/api/database';
+
+export { DocumentSnapshot, QueryDocumentSnapshot } from './src/api/snapshot';
+
+export { SnapshotMetadata } from '../src/api/database';
+
+export {
+  DocumentReference,
+  CollectionReference,
+  Query,
+  doc,
+  collection,
+  collectionGroup,
+  parent
+} from '../lite/src/api/reference';
+
+export { runTransaction, Transaction } from '../lite/src/api/transaction';
```

Comment:
All these exports can be re-used without modification since they don't rely on latency-compensation.